spec/factories/badges.rb

```diff
@@ -4,7 +4,7 @@
   )
 
   factory :badge do
-    title { Faker::Overwatch.quote }
+    title { Faker::Games::Overwatch.quote }
```

Comment:
since you submit a separate PR for faker and fix-db-schema-conflicts this and the other related changes should probably be removed from this PR

Reply:
`foreman` and `faker` use incompatible versions of `thor` so this work has to be based off of #2142.